src/main/scala/nl/biopet/bioconda/BiocondaPlugin.scala

```diff
@@ -0,0 +1,351 @@
+/*
+ * Copyright (c) 2018 Sequencing Analysis Support Core - Leiden University Medical Center
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package nl.biopet.bioconda
+
+import com.typesafe.sbt.GitPlugin
+import com.typesafe.sbt.SbtGit.GitKeys
+import nl.biopet.bioconda.BiocondaDefaults._
+import nl.biopet.bioconda.BiocondaUtils._
+import ohnosequences.sbt.GithubRelease.keys.{TagName, ghreleaseGetRepo}
+import ohnosequences.sbt.SbtGithubReleasePlugin
+import org.kohsuke.github.GitHub
+import sbt.Keys._
+import sbt.{Def, _}
+
+import scala.collection.JavaConverters
+import scala.collection.mutable.ArrayBuffer
+
+object BiocondaPlugin extends AutoPlugin {
+  override def trigger: PluginTrigger = allRequirements
+
+  override def requires: Plugins = {
+    GitPlugin && SbtGithubReleasePlugin
```

Comment:
Assembly?

Reply:
Not needed for this plugin. No keys from assembly are used

Reply:
:+1: